wagtailquickcreate/wagtail_hooks.py

```diff
@@ -1,18 +1,25 @@
 from django.apps import apps
 from django.conf import settings
-from django.conf.urls import url
+from django.urls import path
 from django.utils.safestring import mark_safe
 
-from wagtail.admin.site_summary import SiteSummaryPanel
-from wagtail.core import hooks
+from wagtail.admin.site_summary import SiteSummaryPanel, SummaryItem
+from wagtail import VERSION as WAGTAIL_VERSION
+
+if WAGTAIL_VERSION >= (3, 0):
+    from wagtail import hooks
+else:
+    from wagtail.core import hooks
 
 from .views import QuickCreateView
 
 
-class QuickCreatePanel:
+class QuickCreatePanel(SummaryItem):
+    template_name = "wagtailquickcreate/panel.html"
```

Comment:
`render` doesn't seem applicable anymore, we need to use `template_name` and pass the data via `get_context_data`, otherwise it will raise error saying that `template_name` is missing

Reply:
Looks good